src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -59,10 +61,9 @@ public Uri AppConfigurationEndpoint
 
                     try
                     {
-                        return new Uri(ConnectionStringParser.Parse(_options.ConnectionString, "Endpoint"));
+                        return new Uri(ConnectionStringParser.Parse(_options.ConnectionString, ConnectionStringParser.EndpointSection));
                     }
-                    catch (ArgumentException) { }
-                    catch (UriFormatException) { }

```

Comment:
Who don't we need to catch exceptions by Uri ctor anymore?

[https://docs.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=net-6.0#system-uri-ctor(system-string](https://docs.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=net-6.0#system-uri-ctor(system-string))

Reply:
We are checking `_options.ConnectionString != null` so it won't throw `ArgumentNullException` and `UriFormatException` extends `FormatException` that's why we don't need to explicitly catch `FormatException`.

Reply:
We can do something like this where we won't even have to pass the enumerator down the function and don't have to catch broad exceptions.

![image](https://user-images.githubusercontent.com/1625492/168687713-575fc3a4-c9b2-4fdf-b83b-5c7974c4d4b3.png)